src/util/srp.cpp

```diff
@@ -249,8 +249,7 @@ struct SRPUser {
 
 	char *username;
 	char *username_verifier;
-	unsigned char *password;
-	size_t password_len;
+	unsigned char ucp_hash[SHA512_DIGEST_LENGTH];
```

Comment:
I don't think any changes to this file are necessary at all.
If you make sure to delete the SRP context when done nothing should be left.

Reply:
I can move `srp.cpp` changes to #14129, where I will need to keep this struct alive to allow transfer without storing passwords in string.

Reply:
yes, minimizing the changes in this PR is useful.

Reply:
so moved.

Reply:
Looks like these `srp.cpp` changes will have to be included so as not to break the reconnect function.

Reply:
File changed.